src/data_tools/include/data_tools/benchmark.h

```diff
@@ -70,13 +74,29 @@ struct track_error_benchmark {
     void add_benchmark(std_data::pt_submaps::TransT& trans_corr, std_data::pt_submaps::RotsT& rots_corr, const std::string& name);
     void print_summary();
 
+    // Overloaded functions to work with input submaps in PointsT format
+    void add_ground_truth(PointsT &map_points, PointsT &track_points);
+    void add_benchmark(PointsT &maps_points, PointsT &tracks_points, const std::string &name);
+
+    void track_img_params(PointsT& points_maps, int rows=1000, int cols=1000);
+    cv::Mat draw_height_map(PointsT &points_maps);
+    std::vector<std::vector<std::vector<Eigen::MatrixXd>>> create_grids_from_pings(std_data::mbes_ping::PingsT& pings);
+    std::vector<std::vector<std::vector<Eigen::MatrixXd>>> create_grids_from_matrices(PointsT& points_maps);
+    std::pair<double, Eigen::MatrixXd> compute_consistency_error(
+            std::vector<std::vector<std::vector<Eigen::MatrixXd>>>& grid_maps);
+    cv::Mat draw_error_consistency_map(Eigen::MatrixXd values);
```

Comment:
I think we still need the "compute_draw_error_consistency_map" function for all of our old code to still work? Did you remove that?

Reply:
I split it up into two functions (to reuse some code). afaik that function was only used within add_benchmark(). If so, it isn't needed it anymore.